### DISCUSSION TITLE
CSS variable conflict when used with Quasar

### DISCUSSION THREAD

**Discussion by davidnead** [zouyaoji/vue-cesium#321] [Q&A]
My project is based on Quasar framework (Quasar CLI + Vite).
When I install the vue-cesium components, the global CSS settings of the whole project (theme) change.
This happens because vue-cesium is also built on Quasar, and for some reason vue-cesium's root variables are overriding my project's root variables.

I install the components of vue-cesium Local "on demand". and imports the CSS inside the component like this:
import "vue-cesium/dist/index.css";
As described [here](https://zouyaoji.top/vue-cesium/#/en-US/component/quickstart:~:text=Import%20via%20JavaScript
).

It doesn't matter where I try to import the CSS file, in what order, or in what scope.

Thank you for your help

**Comment by zouyaoji** [comment]
Please refer to [vue-cesium-demo](https://github.com/zouyaoji/vue-cesium-demo) to see if the conflict is still there. If the conflict is still there, please use the latest version of vue-cesium to provide a demo that reproduces the problem, and indicate which css conflicts, I will deal with it.

**Reply by davidnead** [reply]
Thanks. I already tried to install this project earlier, I ran into a problem:
```npm ERR! While resolving: vue-cesium-demo@0.0.0
npm ERR! Found: vite@3.2.4
npm ERR! node_modules/vite
npm ERR!   dev vite@"^3.1.0" from the root project
npm ERR!   peer vite@"^2.0.0 || ^3.0.0" from @quasar/vite-plugin@1.2.3
npm ERR!   node_modules/@quasar/vite-plugin
npm ERR!     dev @quasar/vite-plugin@"^1.2.0" from the root project
```

It should be noted that the Vite version installed globally is 3.2.3
```
vite -v
vite/3.2.3 win32-x64 node-v16.13.0
```
